cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -166,6 +188,8 @@ func (command *Evaluate) Execute(args []string) (err error) {
 		for _, modelID := range command.Models {
 			modelsSelected[modelID] = models[modelID]
 		}
+
+		// TODO Download models in case the provider supports it. https://github.com/symflower/eval-dev-quality/issues/100
```

Comment:
Will remove the TODO. It is in the issue. no need to clutter. Not a priority TODO.